invenio_communities/config.py

```diff
@@ -35,9 +35,12 @@
 COMMUNITIES_DELETE_HOLDOUT_TIME = timedelta(days=365)
 """Time after which the communities marked for deletion are hard-deleted."""
 
-COMMUNITIES_LOGO_EXTENSIONS = ['png', 'jpg', 'jpeg']
+COMMUNITIES_LOGO_EXTENSIONS = ['png', 'jpg', 'jpeg', 'svg']
 """Allowed file extensions for the communities logo."""
 
+COMMUNITIES_LOGO_MAX_SIZE = 1000 * 1000 * 1.5  # 1.5 MB
```

Comment:
``1000 * 1000 * 1.5`` is still valid number number wrt to octets.

If we change it to ``1024^2 * 1.5`` then we should specify (in UI and errors) that the max file size is ``1.5 MiB`` which is less "common" unit than ``1.5 MB``.